inc/class_core.php

```diff
@@ -599,7 +599,41 @@ function __destruct()
  */
 
 $grouppermignore = array("gid", "type", "title", "description", "namestyle", "usertitle", "stars", "starimage", "image");
-$groupzerogreater = array("pmquota", "maxpmrecipients", "maxreputationsday", "attachquota", "maxemails", "maxposts", "edittimelimit", "maxreputationsperuser", "maxreputationsperthread", "emailfloodtime");
```

Comment:
Not a problem with this PR, but I really don't like how 1.8 mixes classes and global variables in a single file 😆

Reply:
Thats why we need to make a list of changes which must be done in next major version of MyBB :)